packages/theme/src/react/index.js

```diff
@@ -19,34 +19,46 @@ export function withTheme(BaseComponent) {
   const name = getDisplayName(BaseComponent)
 
   const Forwarded = React.forwardRef((props, ref) => {
-    const themeName = useTheme()
-    return <BaseComponent {...props} ref={ref} themeName={themeName} />
+    const { enableBleutrals, ...rest } = props
+    const themeName = useTheme({ enableBleutrals })
+    return <BaseComponent {...rest} ref={ref} themeName={themeName} />
   })
   Forwarded.BaseComponent = BaseComponent
   Forwarded.displayName = `withTheme(${name})`
+  Forwarded.propTypes = {
+    enableBleutrals: PropTypes.bool
+  }
+  Forwarded.defaultProps = {
+    enableBleutrals: false
+  }
 
   return hoistNonReactStatics(Forwarded, BaseComponent)
 }
 
-export function useTheme() {
+export function useTheme(config) {
   const themeName = React.useContext(ThemeContext)
-  return themeName
+  const bleutralsAdjustedThemeName =
+    config && config.enableBleutrals ? themeName + 'Bleutrals' : themeName
```

Comment:
Should this be bluetrals if we're going to be doing an entire new color palette?

Reply:
Good point.  This is more than bleutrals.  What is the "new palette" called?  Do we have a name designers are using, @sweeting ?